src/site/content/en/blog/whats-new-pagespeed-insights/index.md

```diff
@@ -0,0 +1,288 @@
+
+### Additional helpful information
+
+The report now includes a new information section at the bottom of each field
+and lab card sharing the following details about the sampled data.
+
++   Data collection period
++   Visit durations
++   Devices
++   Network connections
++   Sample size
++   Chrome versions 
+
+This information should enhance the distinction between lab and field data and
+help users who have previously been uncertain how the two data sources (lab and
+field) might differ. 
+
+{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/jVxqisC0x6I5viUGgjAD.png", alt="enhanced section of information sharing data about field and lab sampling and configuration data", width="800", height="368" %}
+
+### Expand view
+
+We have a new "Expand view" feature that adds a drill-down function to the field
+data section and allows you to view granular details for the Core Web Vitals
+metrics. 
+
+{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/UFVgoK1gJBYk5DLhFwIn.png", alt="ALT_TEXT_HERE", width="800", height="515" %}
+
+### Page image
+
+We have removed the image of the loaded page, which appears right next to the
+field data. The image and thumbnails of the page displaying the loading sequence
+will both be available in the lab data section.
+
+{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/dnIsJA35tj7vs6qgybRM.png", alt="newly expanded view with a drill-down for field data metrics", width="800", height="444" %}
+
+For up to date product documentation, visit
+[https://developers.google.com/speed/docs/insights/.](https://developers.google.com/speed/docs/insights/v5/about)
+
+## Updates to web.dev/measure
+
+To reduce variance between the different tools in our performance toolbox, we
+are also introducing an updated version of
+[web.dev/measure](http://web.dev/measure). As a reminder, PageSpeed Insights is
+a tool for measuring the performance and Core Web Vitals of a page (with lab and
+field data), while web.dev/measure measures page quality more broadly and only
+provides lab data. In the future, Measure will be directly powered by the
+[PageSpeed Insights API](https://developers.google.com/speed/docs/insights/v5/get-started).
+
+Previously, developers would run reports via both the PSI tool and
+[web.dev/measure](http://web.dev/measure) and see different Lighthouse numbers.
+One of the main reasons for differences was because
+[/measure](http://web.dev/measure) originated all tests from the U.S. 
+
+With [web.dev/measure](http://web.dev/measure) using the same API as the PSI UI,
+developers will get a consistent experience when using PSI or
+/[measure](http://web.dev/measure). We have also made a few tweaks to /measure
+based on how users use the tool. As a result, the signed-in experience for
+/measure will be going away, but the most often used part—seeing multiple
+categories—will stay. 
```

Comment:
```suggestion
To reduce variance between the different tools in our performance toolbox, we
are also introducing an updated version of
[web.dev/measure](/measure). As a reminder, PageSpeed Insights is
a tool for measuring the performance and Core Web Vitals of a page (with lab and
field data), while web.dev/measure measures page quality more broadly and only
provides lab data. In the future, Measure will be directly powered by the
[PageSpeed Insights API](https://developers.google.com/speed/docs/insights/v5/get-started).

Previously, developers would run reports via both the PSI tool and
[web.dev/measure](/measure) and see different Lighthouse numbers.
One of the main reasons for differences was because
[/measure](/measure) originated all tests from the U.S. 

With [web.dev/measure](/measure) using the same API as the PSI UI,
developers will get a consistent experience when using PSI or
[web.dev/measure](/measure). We have also made a few tweaks to /measure
based on how users use the tool. As a result, the signed-in experience for
/measure will be going away, but the most often used part—seeing multiple
categories—will stay. 
```